web/web_test.go

```diff
@@ -233,7 +234,7 @@ func TestRoutePrefix(t *testing.T) {
 	go func() {
 		err := webHandler.Run(context.Background())
 		if err != nil {
-			t.Fatalf("Can't start webhandler error %s", err)
+			panic(fmt.Sprintf("Can't start webhandler error %s", err))
```

Comment:
Why are we moving from `t.Fatalf` to `panic` here but do the inverse above?

Reply:
Inside a goroutine you can't use t.Fatalf, but in the main test function you can.